ipn/ipnlocal/local.go

```diff
@@ -6203,3 +6204,157 @@ func mayDeref[T any](p *T) (v T) {
 	}
 	return *p
 }
+
+var errNoSuggestion = errors.New("no suggestion available")
+var errUnableToPick = errors.New("unable to pick candidate")
+
+// SuggestExitNode returns a tailcfg.StableNodeID of a suggested exit node given the local backend's netmap and last report.
```

Comment:
"...returns a tailcfg.StableNodeID, name and tailcfg.Location of a suggested..."